src/common/options/osd.yaml.in

```diff
@@ -149,6 +157,7 @@ options:
   type: float
   level: advanced
   desc: Time in seconds to sleep before next snap trim for HDDs
+  note: This setting is ignored when the mClock scheduler is used.
```

Comment:
```
Python Traceback

yaml.constructor.ConstructorError: found duplicate key

Traceback (most recent call last):
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/common/options/y2c.py", line 366, in 
    main()
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/common/options/y2c.py", line 360, in main
    translate(opts)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/common/options/y2c.py", line 252, in translate
    yml = yaml.load(infile, Loader=UniqueKeySafeLoader)
  File "/usr/lib/python3/dist-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 51, in get_single_data
    return self.construct_document(node)
  File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 60, in construct_document
    for dummy in generator:
  File "/usr/lib/python3/dist-packages/yaml/constructor.py", line 413, in construct_yaml_map
    value = self.construct_mapping(node)
  File "/home/jenkins-build/build/workspace/ceph-pull-requests/src/common/options/y2c.py", line 229, in construct_mapping
    raise yaml.constructor.ConstructorError(None, None,
yaml.constructor.ConstructorError: found duplicate key
```
@nh2 could you drop the duplicated `note`?

Reply:
I got a bit confused; the offending duplicate key is actually 1000 lines further down at line 1266.

It would be good if the `y2c` tool pointed out the location of the duplicate key.

I used `ruamel` to find it:

```python
from ruamel.yaml import YAML; y = YAML().load(open('src/common/options/osd.yaml.in'))
```

It says very nicely where the problem is:

```
ruamel.yaml.constructor.DuplicateKeyError: while constructing a mapping
  in "src/common/options/osd.yaml.in", line 1266, column 3
found duplicate key "note" with value "This setting is automatically reset when the mClock scheduler is used." (original value: "This value is only used if it is non-zero. Normally it is ``0``, which means that the ``hdd`` or ``ssd`` values (below) are used, depending on the type of the primary device backing the OSD.")
  in "src/common/options/osd.yaml.in", line 1278, column 3
```

I pushed a fix.

Reply:
> I pushed a fix.

@tchaikov Please take another look :)